sources/api/migration/migrations/v1.1.5/container-registry-mirrors/src/main.rs

```diff
@@ -0,0 +1,25 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::AddPrefixesMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added a new setting for configuring image registries, `settings.container-registry`
+/// We also added a new configuration template file for the Docker daemon
+fn run() -> Result<()> {
+    migrate(AddPrefixesMigration(vec![
+        "settings.container-registry",
+        "configuration-files.docker-daemon-config",
```

Comment:
Is this value problematic on k8s?

Reply:
No, there is no action done on forward upgrade. On backwards migration, the value is only removed if it exists. Since k8s won't have this set, it's effectively ignored. We are other similar migrations that does the same thing.